app/src/org/commcare/android/framework/CommCareActivity.java

```diff
@@ -236,7 +187,7 @@ private void loadFields() {
                             }
                         }
                         
-                        if(element.locale() != "") {
+                        if(!"".equals(element.locale())) {

```

Comment:
might trigger more than old code, but old code used the java language in a sketchy way.